packages/quantic/package.json

```diff
@@ -63,5 +65,13 @@
     "ts-node": "10.7.0",
     "wait-on": "6.0.1",
     "xml2json": "0.12.0"
-  }
+  },
+  "files": [
+    "force-app/main/default/classes/*",
+    "force-app/main/default/labels/*",
+    "force-app/main/default/lwc/*",
+    "force-app/main/default/cspTrustedSites/*",
+    "force-app/main/default/staticresources/*",
+    "scripts/npm"
+  ]
```

Comment:
Can you show the result of `npm pack` prior to this change and after this change ;)

Reply:
sure!
Before this change it waS basically the whole project:
<img width="493" alt="QuanticPackageBefore" src="https://user-images.githubusercontent.com/86681870/172615405-7770f66f-6a51-46a5-90e9-2454696be63a.png">
After:
<img width="496" alt="quanticPackage" src="https://user-images.githubusercontent.com/86681870/172615437-1edfe4e6-fe0b-4cc7-879b-d3f82e96196b.png">